deps.qt/checksums/qtnetworkauth-everywhere-src-6.3.0.tar.xz.sha256

```diff
@@ -0,0 +1 @@
+203a98942919028520038f90a20cd7ee32b537233545d11e429c3e2c1ad9069f  qtnetworkauth-everywhere-src-6.3.0.tar.xz
```

Comment:
I don't believe that we actually use qtnetworkauth, so you should be able to omit this.